website/react-magma-docs/src/pages/api/theme.mdx

```diff
@@ -109,6 +109,487 @@ export function Example() {
 }
 ```
 
+## Setting up a Theme
+
+To implement theming in your project, we recommend a theme file that contains a theme object that matches React Magma's existing theme structure.
+You'll have to wrap your entire app - or individual components, for more specific theme changes - in the ThemeContext provider from React Magma. Individual
+theme elements can be accessed using context.
+
+A project can have multiple theme files that are triggered conditionally by the Theme Provider.
+
+Custom theming can be applied in the theme file for the following individual components:
+
+- AppBar
+- Combobox
+- Modal
+- Drawer
+- Dropdown
+- Tabs
+- Select
+- Tag
+- Tooltip
+
+The code below is an example of a theme file, including a themes for all valid individual components.
```

Comment:
Should we also include this full theme example in the `Using Themes` example with the dropdown? Wonder if that would be valuable: https://codesandbox.io/s/react-magma-example-forked-gnfp3x?file=/example.tsx